pkg/collector/pipeline.go

```diff
@@ -27,8 +29,17 @@ import (
 // Certain processors might join batches together or split them up.
 type ReportBatch struct {
 	Reports []NelReport
+
 	// When this batch was received by the collector
 	Time time.Time
+
+	// The IP address of the client that uploaded the batch of reports.  You can
+	// typically assume that's the same IP address that was used for the original
+	// requests.
```

Comment:
What's the exact format? Give an example. (Or maybe that's not specified here?)

Reply:
Done.  (Typical string rendering of an IP address)